mmcv/cnn/bricks/non_local.py

```diff
@@ -43,12 +44,16 @@ def __init__(self,
         self.reduction = reduction
         self.use_scale = use_scale
         self.inter_channels = in_channels // reduction
+        if self.in_channels == 0:
+            self.in_channels = 1
```

Comment:
`in_channels == 0` is not a valid input. We may remove this.

Reply:
A typo. It should be `inter_channels`.